visual-migrator/src/app/api/docker/execute-container-command.ts

```
@@ -0,0 +1,360 @@
+    return undefined
+  }
+
+  try {
+    if (command === 'start') {
+      // 1. Start MariaDB container
+      let res: { stdout: string; stderr: string } | Error
+      try {
+        res = await execAsync(
+          `docker run --name ${CONTAINER_NAME} -e MARIADB_ROOT_PASSWORD=\"${DB_PASSWORD}\" -d -p 3306:3306 mariadb:latest`,
+        )
+        const isSuccess = !(res.stderr && res.stderr.trim())
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)